### PR TITLE
1419 skos import

### PR DESCRIPTION
**Pull request by psiotwo**
- SKOS import: [R1419](https://kbss.felk.cvut.cz/redmine/issues/1419)
- SKOS notation to public API: https://github.com/kbss-cvut/termit-ui/issues/163

### CODE REVIEW

File: src/main/java/cz/cvut/kbss/termit/persistence/dao/skos/SKOSImporter.java
Comment (ledsoft) [inline src/main/java/cz/cvut/kbss/termit/persistence/dao/skos/SKOSImporter.java]: Why not use `model.filter(null, RDF.TYPE, SKOS.CONCEPT).stream()?`

Comment (ledsoft) [reply src/main/java/cz/cvut/kbss/termit/persistence/dao/skos/SKOSImporter.java]: Especially given this pattern is used in `setVocabularyLabelFromGlossary`

File: src/main/java/cz/cvut/kbss/termit/persistence/dao/skos/SKOSImporter.java
Comment (ledsoft) [inline src/main/java/cz/cvut/kbss/termit/persistence/dao/skos/SKOSImporter.java]: Hardcoded `/glosář` glossary IRI suffix. What about deployments in other languages?

File: src/main/java/cz/cvut/kbss/termit/persistence/dao/skos/SKOSImporter.java
Comment (ledsoft) [inline src/main/java/cz/cvut/kbss/termit/persistence/dao/skos/SKOSImporter.java]: Commented-out code

File: src/main/java/cz/cvut/kbss/termit/persistence/dao/skos/SKOSImporter.java
Comment (ledsoft) [inline src/main/java/cz/cvut/kbss/termit/persistence/dao/skos/SKOSImporter.java]: Isn't throwing out the whole cache a bit of an overkill?

File: src/main/java/cz/cvut/kbss/termit/service/repository/VocabularyRepositoryService.java
Comment (ledsoft) [inline src/main/java/cz/cvut/kbss/termit/service/repository/VocabularyRepositoryService.java:147]: Here, the parameter order is `rename`, `vocabularyIri` and `file`, but in `SKOSImporter`, the order is `vocabularyIri`, `rename` and `file`-related stuff. IMHO the order of parameters should be consistent.

File: src/test/java/cz/cvut/kbss/termit/persistence/dao/skos/SKOSImporterTest.java
Comment (ledsoft) [inline src/test/java/cz/cvut/kbss/termit/persistence/dao/skos/SKOSImporterTest.java:42]: I don't see any new tests exercising the new _renaming_ functionality.

File: src/test/java/cz/cvut/kbss/termit/util/UtilsTest.java
Comment (ledsoft) [inline src/test/java/cz/cvut/kbss/termit/util/UtilsTest.java]: `model.filter(...).size()`?

**Review by ledsoft — CHANGES_REQUESTED** [review]

File: src/main/java/cz/cvut/kbss/termit/util/Constants.java
Comment (ledsoft) [inline src/main/java/cz/cvut/kbss/termit/util/Constants.java]: I'm sorry but I don't see this as much of an improvement. This will still require changing and recompiling the application just to, for example, switch to English. Why is this not a configurable property just like term or file separator?

**Review by ledsoft — CHANGES_REQUESTED** [review]

File: src/main/java/cz/cvut/kbss/termit/util/Constants.java
Comment (psiotwo) [reply src/main/java/cz/cvut/kbss/termit/util/Constants.java]: I get this, but TBH my intention was just to have it marked somewhere. It is a part of the IRI, not label, so I do not see much value in configuring this at this point.

Comment (ledsoft) [reply src/main/java/cz/cvut/kbss/termit/util/Constants.java]: Ok. I still believe it should be configurable, given the fact that it is really simple to do and it would make it more consistent with existing behavior w.r.t. to term and file IRIs. But after the discussion regarding system configuration I don't think it makes any sense to block this PR on such issue.

**Review by ledsoft — DISMISSED** [review]
Approving with objection.

File: src/main/java/cz/cvut/kbss/termit/util/Constants.java
Comment (psiotwo) [reply src/main/java/cz/cvut/kbss/termit/util/Constants.java]: OK, it is simple, true. Here you are.

**Review by ledsoft — APPROVED** [review]

**Pull request merged**
